### DISCUSSION TITLE
Source string enhancement: replace "Dismiss" with something more clear/neutral like "Close"?

### DISCUSSION THREAD

**Discussion by RockyTDR** [home-assistant/frontend#22845] [Localization]
Could it be considered to replace 'dismiss' with 'close' in the source language? The possible Dutch translations are not always very clear (and very negative), and I believe 'close' fits better,

This is used in e.g. the 'new devices discovered' dialog:

![image-51](https://github.com/user-attachments/assets/928250c2-9723-4ddc-8a59-86e7df7f760c)

Occurrences in frontend: https://app.lokalise.com/project/3420425759f6d6d241f598.13594006/?view=single&reference_lang_id=640&single_lang_id=737&search=dismiss

But also in core: https://app.lokalise.com/project/130246255a974bd3b5e8a1.51616605/?view=multi&search=dismiss
And in Android Companion app: https://app.lokalise.com/project/145814835dd655bc5ab0d0.36753359/?view=multi&search=dismiss

**Comment by NoRi2909** [comment]
I agree, we do have the same issue in German so we generally translate "Dismiss" as "Schließen" ("close") to work around the issue.

It would be much better if the English UI also switched to "Close" so everyone working on HA is aware of the then consistent wording of those buttons across all languages.

**Comment by karwosts** [comment]
I think Close is not the correct word for english. Clicking the button destroys the notification, but close is not a destructive word.

Dismiss fits perfectly IMO (in English).

**Comment by NoRi2909** [comment]
@karwosts Android and Chrome OS both use "Clear" and "Clear all" instead of "Dismiss" for notifications:

Android:

![Screenshot_20241118-171313](https://github.com/user-attachments/assets/4250d9fe-01ff-4cf0-8445-d62615ad2538)

Chrome OS:

![Screenshot 2024-11-18 17 15 51](https://github.com/user-attachments/assets/b5e2512e-fe0d-48f9-95e0-461b85c8dd29)


That's a word that addresses both concerns: it makes clear that you cannot reopen the notification and it creates no such problems in translations.

Google must have had the same discussion with their linguists and settled on that solution. 😄

**Comment by NoRi2909** [comment]
Apple also agrees, it's "Clear" for notifications. This is macOS 15.1 – the latest:

<img width="354" alt="Screenshot 2024-11-18 at 17 48 13" src="https://github.com/user-attachments/assets/2d46fd91-a932-4c29-9e39-5b4deb163346">

Perhaps someone with an iPhone can check on that platform.

**Comment by marcinbauer-ohf** [comment]
Clear is used in the context of all available notifications (group of 1 or N). For a single notification, the common practice is that there is a gesture to "dismiss" an individual one. MD2 doesn't have this so we have to rely on a button, and I agree with @karwosts that the best suited action is still "Dismiss".

We do know the notifications panel needs an overhaul that will have more quality of life improvements than just UX copy. Currently this isn't a priority for the core front-end team, but we can continue this discussion here about this, and steadily prepare a concept and a design.

**Comment by NoRi2909** [comment]
@marcinbauer85 Thanks for the comment. Just FYI here a side-by-side of Google's translations:

![image](https://github.com/user-attachments/assets/d4d105bd-4ec5-411f-bd82-30730eaa6f53)

![image](https://github.com/user-attachments/assets/597578b2-c737-49f0-8bad-32a89e4bd15f)

As you can see they talk about "dismissing" it in the explanation but the buttons are actually "Remove" and "Clear all".

And in the German explanation "Dismiss" turns into "Close" and "Close all" with the buttons then called "Remove" and "Remove all"

So it's no overstatement to say that the use of "dismiss" does create and awful mess in terms of documentation and turns into complete chaos in translations. That's why we as translators ask for getting rid of this word.

In German and Dutch any 1:1 translation of "dismiss" is – pun intended – simply "dismissive" in a very negative way and thus not working at all.

So as much as you might prefer to keep it in English, this will not propagate through many translations at all. And it's always bad for the UX design if what you plan in English will not actually be seen by users in other languages.
